.github/workflows/buildAndTest.yml

```diff
@@ -0,0 +1,40 @@
+
+name: Build and test
+
+defaults:
+  run:
+    shell: bash
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+
+env:
+  BUILD_TYPE: Debug
```

Comment:
Why not in release mode ?

Reply:
Because I need debug info if assert failed during tests.
I have some assert in the kernel code.